### DISCUSSION TITLE
Error Correction

### DISCUSSION THREAD

**Discussion by sydelstan** [ablab/spades#1220] [Q&A] [answered]
### Description of bug

I thought BayesHammer error correction was default, but when I run the following command, I receive a note saying "ONLY assembling (without read error correction)>

### spades.log

[spades.log](https://github.com/ablab/spades/files/13550413/spades.log)

### params.txt

[params.txt](https://github.com/ablab/spades/files/13550422/params.txt)

### SPAdes version

v3.15.5

### Operating System

macOS ventura 13.3

### Method of SPAdes installation

conda

### No errors reported in spades.log

- [X] Yes

**Comment by asl** [comment, marked as answer]
BayesHammer is disabled in isolate mode that expects decently- and uniformly-covered inputs. In this case assembler in the vast majority of cases does not require read error correction.
